src/components/activity-page/text-box/text-box.scss

```diff
@@ -5,9 +5,20 @@
     margin: 0;
   }
   .text-name {
-    position: relative;
-    padding: 4px 8px;
-    clear: both;
-    background-color: $cc-charcoal-light2;
+    font-weight: bold;
+    padding: 8px 0 5px;
+  }
+
+  &.callout {
+    .text-name {
+      background-color: $cc-charcoal-light2;
+      clear: both;
+      position: relative;
+      padding: 8px 20px 5px;
+    }
+    .content {
+      background: #f5f5f5;
```

Comment:
We might want to add this `vars.scss` or, even better, use one of the existing `$cc-charcoal` colors (`$cc-charcoal-light3` is very close).

Reply:
Ah, makes sense. Thanks. I've made that change.